src/main/java/duke/DateManager.java

```diff
@@ -0,0 +1,79 @@
+package duke;
+
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.LocalTime;
+
+public class DateManager {
```

Comment:
Perhaps you can add a short header comment at the start of each class to describe what the class does!